etc/profile-a-l/google-chrome-unstable.profile

```diff
@@ -8,10 +8,16 @@ include globals.local
 noblacklist ${HOME}/.cache/google-chrome-unstable
 noblacklist ${HOME}/.config/google-chrome-unstable
 
+noblacklist ${HOME}/.config/chrome-unstable-flags.conf
+noblacklist ${HOME}/.config/chrome-unstable-flags.config
+
 mkdir ${HOME}/.cache/google-chrome-unstable
 mkdir ${HOME}/.config/google-chrome-unstable
 whitelist ${HOME}/.cache/google-chrome-unstable
 whitelist ${HOME}/.config/google-chrome-unstable
 
+whitelist ${HOME}/.config/chrome-unstable-flags.conf
```

Comment:
Same remarks as above.

Reply:
I've tested the profiles as they are now and google-chrome-unstable does seem to look for `chrome-unstable -flags.conf / chrome-unstable -flags.config`.